docs/api-reference.md

```diff
@@ -218,3 +218,10 @@ Example:
 
     formattedParams.push(connectionHelper.FormatKeyValuePair(driverLocator.keywordDriver, driverLocator.LocateDriver(attr)));
 
+## Deprecated API
+
+### SetImpersonateAttributes connection helper
+This connection helper is deprecated as of Tableau 2020.1, since we always set impersonate attributes for all connectors. Trying to use this in a javascript component will throw an error when attempting to connect.
```

Comment:
```suggestion
This connection helper is deprecated as of Tableau 2020.1, since we always set impersonate attributes for all connectors. Trying to use this in a JavaScript component will throw an error when attempting to connect.
```

Reply:
sorry for the nit, but I _think_ we're standardizing around capitalizing JavaScript like that.